backend/cmd/migrate/migrations/000007_authn_users_create.up.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE authn_users(
+  id text PRIMARY KEY,
+  provider_refresh_token bytea
```

Comment:
Do we not want to store the expiry and the access token here as well?

Reply:
good question. was going to wait on storing the access token for when we plan to use it. right now we are just focused on the silent refresh flow. i haven't quite decided how we're going to store service tokens yet either, e.g. we will feasibly have an okta, github token and maybe more for each user so we will probably have multiple rows for each user for that purpose.

i think i will rename this to `refresh_token` and make a composite primary key with another string field that holds the name of the provider in anticipation of later changes with multiple tokens stored.

the expiry is embedded in the access token's claims so there's no need to store it separately unless it needs to be indexed and range queryable for some purpose.